Add ability of using authorization tokens (#393)
terraform-providers/terraform-provider-vcd: CHANGELOG.md

* Fix code warnings from `staticcheck` and add command `make static` to Travis tests
* `resource/vcd_edge_gateway` and `datasource/vcd_edge_gateway` add `default_external_network_ip`
  field to export default edge gateway IP address - [GH-389]

> * The `vcd` provider now has the ability of connecting through an authorization token - [GH-280]

Reviewer: Thinking how to make this more noticeable, maybe like this:
```suggestion
* Add `token` to the `vcd` provider for an ability of connecting with an authorization token - [GH-280]
```

Author: updated
